models/articles.models.js

```diff
@@ -14,9 +14,30 @@ exports.selectArticleById = (article_id) => {
     });
 };
 
-exports.selectArticles = () => {
+exports.selectArticles = (sort_by, order) => {
   let queryStr =
-    "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url ORDER BY created_at DESC;";
+    "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url";
+
+  const validColumns = ["author", "title", "article_id", "topic", "created_at", "votes", "article_img_url"]
```

Comment:
Think of the categories that a user would want to sort through on the front end - the article_id and article_img_url isn't really of any interest to the user so we shouldn't have these as valid sort-bys.